lib/msal-browser/src/event/EventType.ts

```diff
@@ -8,6 +8,7 @@ export const EventType = {
     INITIALIZE_END: "msal:initializeEnd",
     ACCOUNT_ADDED: "msal:accountAdded",
     ACCOUNT_REMOVED: "msal:accountRemoved",
+    ACTIVE_ACCOUNT_UPDATED: "msal:activeAccountChanged",
```

Comment:
Updated or Changed? Let's be consistent over both left and right hand sides

Reply:
Any preferences? I'll make sure it's standardized.

Reply:
I missed this.  Changed might be clearer but I'm ok either way.

Reply:
Updated the event name to "changed." I picked ACTIVE_ACCOUNT_UPDATED and copilot completed the line. I intended to change it to "updated" but forgot about it. I think "changed" is better as well.